### PR TITLE
Add a reminder of the finals details

### PR DESCRIPTION
**Pull request by PeterJCLaw**
There's nothing new here, just a condensed reminder of important information for the teams.

Fixes https://github.com/srobo/tasks/issues/684

### CODE REVIEW

File: SR2021/2021-04-27-finals-reminder.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[feedback survey][feedback-survey] before the knockouts.
+
+## Code Submission Deadline
+
+The deadline for [code submission][code-submitter] for the knockouts is 8pm BST
+(UK time) on Friday the 30th April. If your team is going to have any difficulty
+uploading your code by this time let us know before the deadline and we'll try
+to sort something out.
+
+*Note*: if you don't submit code for the knockouts we'll use the code you
```
Comment (PeterJCLaw) [inline SR2021/2021-04-27-finals-reminder.md:26]: Should this explicitly say that teams who didn't enter league 4 aren't eligible? Or should we drop this section out for the ineligible teams? The latter feels nicer, if it's easy enough to do.

Comment (RealOrangeOne) [reply SR2021/2021-04-27-finals-reminder.md:26]: Teams who didn't enter league 4 aren't in the knockouts anyway, so feels unnecessary to be explicit.

File: SR2021/2021-04-27-finals-reminder.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Now that the league stage is over, we wanted to give you some quick reminders
+about the schedule for next weekend's knockouts. We'd also love to hear about
+about your experiences competing in SR2021.
+
+## Feedback & Demographics Survey
+
+To help us understand how we're doing we'd really like to understand the make up
+of your team and their experiences with the competition this year. If they
+haven't done so already please encourage your competitors to complete our
```
Comment (RealOrangeOne) [inline SR2021/2021-04-27-finals-reminder.md:16]: Given this is aimed at adults, perhaps adding something explicit like it helps with funding and understanding how we can best help competitors? Then team leaders are more likely to want to get competitors to complete it.

File: SR2021/2021-04-27-finals-reminder.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+to: All SR2021 teams
+subject: Student Robotics Finals
```
Comment (RealOrangeOne) [inline SR2021/2021-04-27-finals-reminder.md:3]:
```suggestion
subject: Student Robotics Finals Details
```

**Review by RealOrangeOne — DISMISSED** [review]

File: SR2021/2021-04-27-finals-reminder.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2021/2021-04-27-finals-reminder.md:26]: Yes, we know that. However it doesn't stop them submitting code and thinking that they'll get entered.

File: SR2021/2021-04-27-finals-reminder.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2021/2021-04-27-finals-reminder.md:16]: Good idea. Could you suggest some wording to that effect?

File: SR2021/2021-04-27-finals-reminder.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2021/2021-04-27-finals-reminder.md:26]: They know it too - it was in the [previous email](https://github.com/srobo/team-emails/blob/master/SR2021/2021-04-20-league-4-knockouts-simulator-update.md)

File: SR2021/2021-04-27-finals-reminder.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Now that the league stage is over, we wanted to give you some quick reminders
+about the schedule for next weekend's knockouts. We'd also love to hear about
+about your experiences competing in SR2021.
+
+## Feedback & Demographics Survey
+
+To help us understand how we're doing we'd really like to understand the make up
+of your team and their experiences with the competition this year. If they
+haven't done so already please encourage your competitors to complete our
+[feedback survey][feedback-survey] before the knockouts.
```
Comment (RealOrangeOne) [inline SR2021/2021-04-27-finals-reminder.md:17]:
```suggestion
To help us understand how we're doing we'd really like to understand the make up
of your team and their experiences with the competition this year. If they
haven't done so already please encourage your competitors to complete our
[feedback survey][feedback-survey] before the knockouts. These feedback forms are vital
both in securing funding, and ensuring we're always improving our events so competitors
get the most out of them they possibly can.
```

File: SR2021/2021-04-27-finals-reminder.md (hunk 2 above)
Comment (RealOrangeOne) [reply SR2021/2021-04-27-finals-reminder.md:16]: https://github.com/srobo/team-emails/pull/83/files#r620966282

File: SR2021/2021-04-27-finals-reminder.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2021/2021-04-27-finals-reminder.md:26]: True. Repetition is good however.

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**